CenterForOpenScience/cos.io
# [EOSF-812] Add Emerald Publishing journals

## Ticket

https://openscience.atlassian.net/browse/EOSF-812

## Purpose

To add Emerald Publishing journals to the database

## Changes

- Created json file with journal data
- Added migration to put data in the database

## QA

- To test this on staging, go to this url: https://staging-cosio.herokuapp.com/our-services/top-guidelines/
- Click on the `Current Signatories` tab on the side navigation to get to the journals table.
- In the search bar, type `emerald` to search for Emerald Publishing
- Check that the journals show up that have Emerald Publishing as their publisher

There shouldn't be any other information in the table for each entry besides the title and the publisher.
